### ISSUE TITLE
Swift support

### ISSUE THREAD

**Issue by madnik** [rubychan/coderay#185]
You have plan to introduce swift support.

**Comment by korny** [comment]
Eventually, yes. Right now, Swift syntax is in flux, and I'm not currently developing new scanners. You can try [writing your a scanner on your own](https://github.com/rubychan/coderay/wiki/Scanner-Requests) if you feel adventurous :)

**Comment by CrockCity** [comment]
+1

**Comment by adamdahan** [comment]
@korny any update on this? I am sure Swift is syntactically stable now

**Comment by korny** [comment]
I agree that it would be the right time to add Swift, but I can't find the time to write new Scanners at the moment. So, I hope somebody else can write one…or you wait until I'm finished with CodeRay 2, which hopefully makes writing scanners easier :P
